### PR TITLE
Addition of introduction page

### PR DESCRIPTION
**Pull request by nathancashmoredwp**

### CODE REVIEW

File: app/views/introduction.mustache [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{<layout}}
+  {{$pageTitle}}{{#__}}introduction.pageTitle{{/__}}{{/pageTitle}}
+  {{$applicationContent}}
+    <h2 class="heading-xlarge">{{#__}}introduction.header{{/__}}</h2>
+
+
+    <p>Use this tool to help with your work search.</p>
```
Comment (lukasz-kaniowski) [inline app/views/introduction.mustache]: RYWS has all the copy in `en.json` file.

File: app/views/introduction.mustache [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    <ol class="list list-number">
+      <li>Build a list of jobs you're interested in.</li>
```
Comment (lukasz-kaniowski) [inline app/views/introduction.mustache]: Same as above

Comment (nathancashmoredwp) [inline app/views/introduction.mustache]: See updates below.

**Comment by lukasz-kaniowski** [comment]
👍

**Pull request merged**